### PR TITLE
#79 Prevents lightchain node dir to be overwritten accidentally on init.

### PR DESCRIPTION
**Pull request by EnchanterIO**
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests

### CODE REVIEW

File: cmd/lightchain/init.go [hunk 1]
```diff
@@ -75,6 +76,13 @@ func newNodeCfgFromCmd(cmd *cobra.Command) (node.Config, setup.Network, error) {
 	shouldTrace, _ := cmd.Flags().GetBool(TraceFlag.Name)
 	traceLogFilePath, _ := cmd.Flags().GetString(TraceLogFlag.Name)
 
+	// This should be done inside of the `node.Init()` pkg but due to bad design,
```
Comment (ggarri) [inline cmd/lightchain/init.go]: Would you mind to create an issue and include the number as part of this doc?

File: database/node.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ func NewNode(cfg *Config, consensusAPI conAPI.API, registry *prometheus.Registry
 		return nil, err
 	}
 
+	// Todo: Should be refactored, creating a new instance of a struct SHOULDN'T do any FS changes
```
Comment (ggarri) [inline database/node.go]: This comment could be part of the issue added to the backlog refering to the pending refactor

File: cmd/lightchain/init.go (hunk 1 above)
Comment (EnchanterIO) [reply cmd/lightchain/init.go]: done https://github.com/lightstreams-network/lightchain/pull/88/commits/802bf83493a760dae603bcba3875780755e846ac

File: database/node.go (hunk 2 above)
Comment (EnchanterIO) [reply database/node.go]: done https://github.com/lightstreams-network/lightchain/pull/88/commits/802bf83493a760dae603bcba3875780755e846ac

File: cmd/lightchain/init.go (hunk 1 above)
Comment (EnchanterIO) [reply cmd/lightchain/init.go]: done https://github.com/lightstreams-network/lightchain/pull/88/commits/802bf83493a760dae603bcba3875780755e846ac

**Pull request merged**